use-timescale/fork-services.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Developer service forks
+excerpt: With developer forks, spin up a branch, run your test, throw it away, or merge it back.
+products: [cloud]
+price_plans: [performance, scale, enterprise]
+keywords: [backups, restore]
+tags: [recovery, failures]
+---
+
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
+import CLIFORKS from "versionContent/_partials/_devops-cli-service-forks.mdx";
+
+# Developer $SERVICE_SHORT forks
+
+Modern development is highly iterative. Developers and AI agents need safe spaces to test changes before deploying them
+to production. Forkable $SERVICE_SHORTs make this natural and easy. Spin up a branch, run your test, throw it away, or
+merge it back.
+
+A fork is an exact copy of a $SERVICE_SHORT at a specific point in time, with its own independent data and configuration,
+including:
+- The database data and schema
+- Configuration
+- An admin `tsdbadmin` user with a new password
+
+Forks are fully independent. Changes to the fork don't affect the parent $SERVICE_SHORT, you can query
+them, run migrations, add indexes, or test new features.
+
+Forks are a powerful way to share production-scale data safely. BI and data science teams often need access to real
+datasets to build models or generate insights. With forkable $SERVICE_SHORTs, you easily create instant, zero-copy
```

Comment:
I think we [changed to wording](https://iobeam.slack.com/archives/C07JSQ7SL00/p1760568749145069?thread_ts=1760568728.161159&cid=C07JSQ7SL00) from "instant" to "fast"

```suggestion
datasets to build models or generate insights. With forkable $SERVICE_SHORTs, you easily create fast, zero-copy
```